examples/ci-tests/create-and-destroy-aws/environment.yml

```diff
@@ -1,6 +1,6 @@
 # This opta environment is used by the test-ci job in the runxc repo.
-name: opta-ci
-org_name: runx
+name: awsenv-ci
+org_name: runx-ci
```

Comment:
Can we keep this runx so we have proper accounting in amplitude

Reply:
Sure, I'll make this change.